server/apps/archive/archive_spike.py

```diff
@@ -69,9 +70,17 @@ class ArchiveSpikeService(BaseService):
 
     def on_update(self, updates, original):
         updates[ITEM_OPERATION] = ITEM_SPIKE
+        self._validate_item(original)
         self._validate_take(original)
         self._update_rewrite(original)
 
+    def _validate_item(self, original):
+        # only allow an item to be spiked if it is not a member of a package
+        if original[ITEM_TYPE] != CONTENT_TYPE.COMPOSITE and original.get(LINKED_IN_PACKAGES, None) \
+                and len([x for x in original.get(LINKED_IN_PACKAGES, []) if x.get(PACKAGE_TYPE, '') == '']):
```

Comment:
How long is this list expected to be? Probably pretty short, since it only contains the linked packages?

If you think it's needed, we can get rid of the temporary list by saying `sum(1 for x in ...)`.